website/lib/main.dart

```diff
@@ -51,6 +52,13 @@ final GoRouter _router = GoRouter(
             return const PrivacyPolicyPage();
           },
         ),
+        if (isSharezonePlusPageEnabledFlag)
+          GoRoute(
+            path: SharezonePlusPage.tag,
+            builder: (BuildContext context, GoRouterState state) {
+              return const SharezonePlusPage();
+            },
+          ),
```

Comment:
Just a button for us to access the plus page. It's likely that we change this later.